tests/foreman/api/test_provisioning.py

```diff
@@ -179,6 +179,8 @@ def test_rhel_pxe_provisioning(
         assert provisioning_host.execute('test -s /mnt/sysimage/root/install.post.log').status == 1
 
         # Run a command on the host using REX to verify that Satellite's SSH key is present on the host
+        # Add workaround for SAT-32007 and SAT-32006
```

Comment:
Why not add if is_open condition for the command?

Reply:
Added `is_open` condition